packages/java/endpoint/src/main/java/com/vaadin/hilla/springnative/HillaHintsRegistrar.java

```diff
@@ -40,8 +39,6 @@ public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
         registerEndpointTypes(hints);
 
         hints.resources().registerPattern("file-routes.json");
-        hints.reflection().registerType(ClientViewConfig.class,
-                MemberCategory.values());
```

Comment:
Reminder: This also needed to be verified separately/manually by running a native build for the `[hilla-project]/tests/spring/native` project, as it is not part of the validation build workflow ATM.

Reply:
So what is required for a native build execution?
Has it been tested on windows? I just keep getting `cannot find module` issues.

Reply:
Didn't have any issues with the native apart from the old node version.

Reply:
If the native build and running the image was successful, this can be considered as resolved.